src/sqlite_manager.py

```diff
@@ -0,0 +1,74 @@
+import sqlite3
+import pandas as pd
+import os
+
+
+class SQLiteManager:
+    def __init__(self, db_file):
+        self.db_file = db_file
+        self.conn = None
+        self.create_connection()
+
+    def create_connection(self):
+        try:
+            self.conn = sqlite3.connect(self.db_file)
+            print("Connected to SQLite database.")
```

Comment:
You have a configured logger, it would be better to reuse it also for such cases